conformance/provisioner/static-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+---
+# Source: nginx-kubernetes-gateway/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-gateway
+  namespace: nginx-gateway
+  labels:
+    app.kubernetes.io/name: nginx-gateway
+    app.kubernetes.io/instance: nginx-gateway
+    app.kubernetes.io/version: "0.5.0"
+spec:
+  # We only support a single replica for now
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-gateway
+      app.kubernetes.io/name: nginx-gateway
+      app.kubernetes.io/instance: nginx-gateway
+  template:
+    metadata:
+      labels:
+        app: nginx-gateway
```

Comment:
Sorry, I know it's not related to this change, but I just noticed it now... should we remove `app: nginx-gateway`? Seems redundant since we have `app.kubernetes.io/name: nginx-gateway`

Reply:
we don't use it in the generated Service

Reply:
Removed 🔥